src/views/Login.scss

```diff
@@ -8,17 +8,17 @@
     height: 100vh;
 }
 
+.login-image {
+    background-color: #fff;
+    background-image: url('../images/LoginImage.svg');
+    background-position: center;
+    background-repeat: no-repeat;
+}
+
 .right-section {
     color: #fff;
     height: 100vh;
     position: relative;
-    .form {
-        align-items: center;
-        display: flex;
-        height: 100vh;
-        margin: auto;
-        width: 50%;
-    }
```

Comment:
These had the intention of vertically centering the forms. With just the component's margin I cannot see the bottom part of the form in my machine